theseus/extlib/tests/test_cusolver_lu_solver.py

```diff
@@ -23,8 +23,10 @@ def check_lu_solver(
         return
     from theseus.extlib.cusolver_lu_solver import CusolverLUSolver
 
+    rng = torch.Generator()
+    rng.manual_seed(0)
```

Comment:
Any reason the manual_seed is hash based from params in baspacho test but set to default in lu test?

Reply:
We could do the same thing here. I was just maintaining consistency with the original code. I'm guessing @maurimo hadn't thought of the hash version before baspacho.